corehq/apps/user_importer/importer.py

```diff
@@ -335,11 +335,21 @@ def create_or_update_users_and_groups(domain, user_specs, group_memoizer=None, u
                             'Changing usernames is not supported: %(username)r to %(new_username)r'
                         ) % {'username': user.username, 'new_username': username})
 
+                    # note: explicitly not including "None" here because that's the default value if not set.
+                    # False means it was set explicitly to that value
+                    if is_account_confirmed is not False:
```

Comment:
a bit confused by this. This is under the branch for updating existing users but the error message appears to be for new users. Also won't this error if you leave the column blank?

Reply:
If you leave the column blank I'm pretty sure it will be `None` (I thought there was a test for this - will double check/confirm tomorrow).

But you're right this should be `is False`. I broke it (and the tests) on the last stickler commit. fixing now. https://github.com/dimagi/commcare-hq/pull/26864/commits/60cbfe38c5631e5a1ea326392379aff7cb1aac9c

Reply:
https://github.com/dimagi/commcare-hq/pull/26864/commits/64e1d9de6f31bef106e4122d2fe10c47b8de5f46

Reply:
confirmed that leaving the column blank works fine but there's not a test. I'll add it in the next PR

Reply:
https://github.com/dimagi/commcare-hq/pull/26888